src/main/typescript/lib/formComponents/Checkbox.tsx

```diff
@@ -16,6 +16,8 @@
 import * as React from "react"
 import { WrappedFieldProps } from "redux-form"
 
+// foo is used here to make sure it all compiles
+// if we leave it out, the <Field component={Checkbox}/> will fail to compile
```

Comment:
thanks for the explanation. should we rename to `satisfyTheCompiler` or some such?